src/pages/home/Home.tsx

```diff
@@ -7,15 +7,18 @@ import { useFetch } from "hooks/useFetch";
 import { Route } from "routes/index.lazy";
 
 export default function Home() {
-  const bookFallbacks = Array.from({ length: 20 }, (_, i) => <BookFallback key={i} />);
+  const bookFallbacks = Array.from({ length: 20 }, (_, i) => (
+    <BookFallback key={i} />
+  ));
```

Comment:
wystaw to poza komponent, nie ma sensu, żeby przy każdym rerenderze na nowo fallbacki się tworzyły

np. gdy dane się załadują to na nowo tworzysz 20 instancji komponentów fallback, gdy ktoś zmieni search, lub cokolwiek innego